.circleci/config.yml

```diff
@@ -1,11 +1,12 @@
 version: 2.1
 orbs:
-  architect: giantswarm/architect@0.8.5
+  architect: giantswarm/architect@0.16.0
```

Comment:
any reason why you don't want to use the latest? `1.1.1` is available.

Reply:
architect? did not know, based it on efk stack app

Reply:
where i can check the latest for architect? i see version 3x smth in the repo

Reply:
> Find the latest ARCHITECT_ORB_VERSION here: https://circleci.com/orbs/registry/orb/giantswarm/architect, then update the CircleCi config with it.

as described in https://intranet.giantswarm.io/docs/dev-and-releng/app-developer-processes/adding_app_to_appcatalog/ ;)

Reply:
done